weblate/api/tests.py

```diff
@@ -215,6 +215,42 @@ def test_add_group(self):
             request={"group_id": group.id},
         )
 
+    def test_remove_group(self):
+        group = Group.objects.get(name="Viewers")
```

Comment:
To make the code more readable, I'd fetch the username just once instead of doing that with every test:

```suggestion
        group = Group.objects.get(name="Viewers")
        username = User.objects.filter(is_active=True).first().username
```